galaxy_importer/config.py

```diff
@@ -0,0 +1,66 @@
+# (c) 2012-2020, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import configparser
+import os
+
+
+FILENAME = 'galaxy-importer.cfg'
+
+
+class Config(object):
```

Comment:
Could the config be in yaml? May make it easier for other tools to generate/modify the config file (write support for ini files is sometimes iffy...). Also fits in the kubernetes/openshift approach. ansible (and ansible-test) itself uses a random mixmash of ini, yaml, json, etc so ini wouldn't be out of place. 

But probably doesn't really matter. Especially if the config options are small. And more so if the loading/serializing and config instance get decoupled where it would be :v:easy:v: (airquote) to change the format.